src/elmWorkspaceSymbols.ts

```diff
@@ -1,99 +1,96 @@
 import * as vscode from 'vscode';
 
-import { TextDocument } from 'vscode';
+import { TextDocument, SymbolInformation } from 'vscode';
 import { processDocument } from './elmSymbol';
 
 const config = vscode.workspace.getConfiguration('elm');
 
 export class ElmWorkspaceSymbolProvider
```

Comment:
This provider could expose functionality to get parsed modules for use in other places e.g. elmDelphi.